aim/web/ui_v2/src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -0,0 +1,56 @@
+import { Breadcrumbs, Button, Grid, Link, Typography } from '@material-ui/core';
+import { ChevronLeft, ChevronRight } from '@material-ui/icons';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import React from 'react';
+
+function BreadCrumb(): React.FunctionComponentElement<React.ReactNode> {
```

Comment:
@aimhubio/uidevs this component should be renamed to smth like AppNavbar, shouldn't it?